iib/workers/tasks/utils.py

```diff
@@ -23,6 +24,139 @@
 dogpile_cache_region = create_dogpile_region()
 
 
+def deprecate_bundles(
+    bundles, base_dir, binary_image, from_index, overwrite_target_index_token=None
```

Comment:
can we add each attribute on separate line?

Reply:
Sorry, it's done by python-black auto-formatter :)

Reply:
no problem